README.md

```diff
@@ -40,16 +40,58 @@ The exporter collects a number of statistics from the server:
 # TYPE memcached_current_connections gauge
 # HELP memcached_current_items Current number of items stored by this instance.
 # TYPE memcached_current_items gauge
-# HELP memcached_items_evicted_total Number of valid items removed from cache to free memory for new items.
+# HELP memcached_items_evicted_total Total number of valid items removed from cache to free memory for new items.
 # TYPE memcached_items_evicted_total counter
-# HELP memcached_items_reclaimed_total Number of times an entry was stored using memory from an expired entry.
+# HELP memcached_items_reclaimed_total Total number of times an entry was stored using memory from an expired entry.
 # TYPE memcached_items_reclaimed_total counter
 # HELP memcached_items_total Total number of items stored during the life of this instance.
 # TYPE memcached_items_total counter
 # HELP memcached_limit_bytes Number of bytes this server is allowed to use for storage.
 # TYPE memcached_limit_bytes gauge
+# HELP memcached_malloced_bytes Number of bytes of memory allocated to slab pages.
+# TYPE memcached_malloced_bytes gauge
 # HELP memcached_read_bytes_total Total number of bytes read by this server from network.
 # TYPE memcached_read_bytes_total counter
+# HELP memcached_slab_chunk_size_bytes Number of bytes allocated to each chunk within this slab class.
+# TYPE memcached_slab_chunk_size_bytes gauge
+# HELP memcached_slab_chunks_free Number of chunks not yet allocated items.
+# TYPE memcached_slab_chunks_free gauge
+# HELP memcached_slab_chunks_free_end Number of free chunks at the end of the last allocated page.
+# TYPE memcached_slab_chunks_free_end gauge
+# HELP memcached_slab_chunks_per_page Number of chunks within a single page for this slab class.
+# TYPE memcached_slab_chunks_per_page gauge
+# HELP memcached_slab_chunks_total Total number of chunks allocated to this slab class.
```

Comment:
I'm still unsure whether this is a counter. While the original `total_chunks` naming could mean that, I have the feeling this is actually a gauge representing the current amount of chunks used. A common naming scheme in Prometheus would be `memcached_slab_current_chunks`. Though, given the ambiguity of the original metric name, let me check some more of our servers.

The latest memcached versions support automatic slab rescaling, I'm trying to find out whether that would cause such numbers to go down again.